module-2/backend/week_one.md

```diff
@@ -5,30 +5,83 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  GET - show a record from the database
+  POST - create a record in the database
+  PUT - update a record in the database
+  DELETE - remove a record from database
+
 2. What is Sinatra?
+  a framework that starts a server, and makes it simple to manage routes
+
 4. What is MVC?
+  convention from rails that is a standard template for organizing web app paths

```

Comment:
👍 One note: you will see this pattern in places besides Rails, but this is the right idea.